### DISCUSSION TITLE
Не могу ввести ключ, как это сделать, помогите пожалуйста.

### DISCUSSION THREAD

**Discussion by stepantu** [danny-avila/LibreChat#3147] [Q&A]
Не могу ввести ключ, как это сделать, помогите пожалуйста.

**Comment by prononext** [comment]
Если вы дадите нам более подробную информацию, мы сможем вам помочь. Возможно, вы также можете использовать Google Translator для своих запросов на английском языке. Так вы быстрее получите помощь. @stepantu
